### PR TITLE
Exclude usage moving forward

### PR DESCRIPTION
**Pull request by chrisronline**
Relates to https://github.com/elastic/kibana/issues/83626

This needs to be in 7.11 and master

### CODE REVIEW

File: playbooks/monitoring/kibana/docs_compare.py [hunk 1]
```diff
@@ -30,29 +30,27 @@ def handle_special_case_kibana_settings(legacy_doc, metricbeat_doc):
...
-  metricbeat_doc["kibana_stats"]["usage"]["search"]["total"] = legacy_doc["kibana_stats"]["usage"]["search"]["total"]
-  metricbeat_doc["kibana_stats"]["usage"]["search"]["averageDuration"] = legacy_doc["kibana_stats"]["usage"]["search"]["averageDuration"]
-
   # Special case for https://github.com/elastic/kibana/pull/76730
   # To be removed if/when https://github.com/elastic/beats/issues/21092 is resolved
   metricbeat_doc["kibana_stats"]["os"]["cpuacct"] = legacy_doc["kibana_stats"]["os"]["cpuacct"]
   metricbeat_doc["kibana_stats"]["os"]["cpu"] = legacy_doc["kibana_stats"]["os"]["cpu"]
 
-def filter_kibana_usage_stats(legacy_doc, metricbeat_doc):
-  for i in unused_kibana_usage_properties:
```
Comment (igoristic) [inline playbooks/monitoring/kibana/docs_compare.py:43]: We should then also remove `unused_kibana_usage_properties` hoisted at the top

File: playbooks/monitoring/kibana/docs_compare.py [hunk 2]
```diff
@@ -30,29 +30,27 @@ def handle_special_case_kibana_settings(legacy_doc, metricbeat_doc):
...
+    
 
 def handle_special_cases(doc_type, legacy_doc, metricbeat_doc):
     if doc_type == "kibana_settings":
         handle_special_case_kibana_settings(legacy_doc, metricbeat_doc)
     if doc_type == "kibana_stats":
+        # Starting in 7.11, we want to stop storing `usage` data within Kibana monitoring documents
+        # https://github.com/elastic/kibana/issues/83521
+        legacy_doc["kibana_stats"]["usage"] = False
+        metricbeat_doc["kibana_stats"]["usage"] = False
```
Comment (igoristic) [inline playbooks/monitoring/kibana/docs_compare.py:47]: Maybe we should just delete it instead?
```suggestion
        del legacy_doc["kibana_stats"]["usage"]
        del metricbeat_doc["kibana_stats"]["usage"]
```

**Review by igoristic — COMMENTED** [review]
Just some minor changes/preferences/nit mentioned in my comments, but other than that I think it looks good

File: playbooks/monitoring/kibana/docs_compare.py (hunk 2 above)
Comment (chrisronline) [reply playbooks/monitoring/kibana/docs_compare.py:47]: Yea, this is better. I'll do that

File: playbooks/monitoring/kibana/docs_compare.py (hunk 1 above)
Comment (chrisronline) [reply playbooks/monitoring/kibana/docs_compare.py:43]: Nice, good find

**Review by igoristic — APPROVED** [review]
Looks good! 👍

**Pull request merged**